### PR TITLE
implement isWindowVisible

### PR DESCRIPTION
**Pull request by ahkohd**
Hi, @sentialx  I implemented isWindowVisible method to check if a window is visible (win32). I need this feature in my project. Thanks, hope you merge this and release a new version on NPM soonest. Am depending on it.

https://superuser.com/questions/1332099/windows-7-10-command-line-show-running-applications-windows
https://devblogs.microsoft.com/oldnewthing/?p=97606
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-iswindowvisible

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,9 @@ Returns `number` - monitor handle.
 
 Returns `boolean` - whether the window is a valid window.
 
+#### win.isWindowVisible() `Windows`
```
Comment (sentialx) [inline README.md]: I think this function should be named `isVisible`.

**Review by sentialx — CHANGES_REQUESTED** [review]

**Comment by ahkohd** [comment]
I have made the changes.

**Comment by sentialx** [comment]
Thanks for your PR!

**Comment by ahkohd** [comment]
Awesome! waiting for the NPM update.

**Pull request merged**